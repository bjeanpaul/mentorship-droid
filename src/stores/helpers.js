import moment from 'moment';
import { values, isUndefined, sortBy, filter, first } from 'lodash';


export const getAuthUserProfile = ({
  auth: { profileId },
  entities: { profiles },
}) => !isUndefined(profileId)
  ? profiles[profileId]
  : null;


export const getContext = state => {
  const { auth: { auth } } = state;

  return {
    auth,
    profile: getAuthUserProfile(state),
  };
};


// TODO sort?
export const getCategories = ({ entities: { categories } }) => values(categories);


// TODO sort?
export const getCategoryActivities = ({
  entities: { activities },
}, targetCategoryId) => (
  values(activities)
    .filter(({ category }) => category === targetCategoryId));


export const getCategory = ({ entities: { categories } }, id) => categories[id];


export const getActivity = ({ entities: { activities } }, id) => activities[id];


export const getScheduledCalls = ({ entities: { scheduledCalls } }) => values(scheduledCalls);


export const getScheduledCall = ({ entities: { scheduledCalls } }, id) => scheduledCalls[id];


export const getScheduledCallActivity = (state, id) => {
  const scheduledCall = getScheduledCall(state, id);
  return scheduledCall && scheduledCall.activity && getActivity(state, scheduledCall.activity);
};


export const getEvents = ({ entities: { events } }) => values(events);


export const getActivityCallNotes = ({
  entities: { callNotes },
}, activityId) => (
  filter(callNotes, { callActivity: activityId }));


export const getCall = ({ entities: { calls } }, id) => calls[id];


export const getCallNote = ({ entities: { callNotes } }, id) => callNotes[id];


export const getNextScheduledCall = (state, predicate = {}, time = Date.now()) => {
  let res = sortBy(getScheduledCalls(state), ({ callTime }) => callTime);
  res = res.filter(({ callTime }) => moment(callTime).isSameOrAfter(time));
  res = filter(res, predicate);
  return first(res);
};
