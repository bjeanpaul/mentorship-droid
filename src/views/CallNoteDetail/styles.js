import { StyleSheet } from 'react-native';
import colors from 'src/constants/colors';

export default StyleSheet.create({
  scrollView: {
    padding: 24,
  },
  titleContainer: {
    borderColor: colors.CALL_NOTE_DETAIL_TITLE_BORDER,
    borderBottomWidth: StyleSheet.hairlineWidth,
    borderStyle: 'solid',
    paddingBottom: 19,
    marginBottom: 18,
  },
  section: {
    marginBottom: 45,
  },
  sectionBody: {
  },
  sectionBodyActivityItems: {
    flex: 1,
    flexDirection: 'row',
    alignItems: 'center',
  },
  activityImage: {
    width: 72,
    height: 72,
    marginRight: 16,
    marginBottom: 12,
  },
  menteeStateImage: {
    alignSelf: 'center',
  },
  objectiveAchievedImage: {
    alignSelf: 'center',
  },
  activityObjective: {
    flex: 1,
    paddingTop: 0,
    paddingBottom: 0,
  },
  activityHelpfulText: {
    textAlign: 'center',
  },
});
