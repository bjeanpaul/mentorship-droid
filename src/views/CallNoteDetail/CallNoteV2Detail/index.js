import moment from 'moment';
import { fromPairs, upperFirst } from 'lodash';

import React, { PropTypes } from 'react';
import { View, ScrollView, Image } from 'react-native';
import { BaseView, Header, HeaderIcon, Text } from 'src/components';
import * as constants from 'src/constants/callNotes';
import images from 'src/constants/images';
import styles from './styles';


const mentee = fromPairs([
  [constants.CALL_NOTES_MENTEE_HAPPY, images.CALL_NOTES_MENTEE_HAPPY],
  [constants.CALL_NOTES_MENTEE_SAD, images.CALL_NOTES_MENTEE_SAD],
  [constants.CALL_NOTES_MENTEE_BORED, images.CALL_NOTES_MENTEE_BORED],
  [constants.CALL_NOTES_MENTEE_CONFUSED, images.CALL_NOTES_MENTEE_CONFUSED],
  [constants.CALL_NOTES_MENTEE_UPSET, images.CALL_NOTES_MENTEE_UPSET],
  [constants.CALL_NOTES_MENTEE_WITHDRAWN, images.CALL_NOTES_MENTEE_WITHDRAWN],
]);

// const activity_completion =


// EXCELLENT', 'Excellent'
// OK', 'Ok'
// INAUDIBLE', 'Inaudible'
// DROPPED', 'Dropped'
// DELAYED', 'Delayed'

const callResults = fromPairs([
  ['COMPLETED', 'We completed it'],
  ['PARTIALLY_COMPLETED', 'Partially completed'],
  ['MENTEE_NOT_AVAILABLE', 'Mentee not available'],
  ['MENTEE_RESCHEDULED', 'Mentee rescheduled'],
]);


const objectAchievedOptions = fromPairs([
  ['1', 'Objective was achieved'],
  ['2', 'Mostly achieved'],
  ['3', 'Somewhat achieved'],
  ['4', 'Achieved a little'],
  ['5', 'Not at all'],
]);

const activityRatingOptions = fromPairs([
  ['1', 'Very good'],
  ['2', 'Good'],
  ['3', 'Average'],
  ['4', 'Poor'],
  ['5', 'Very Poor'],
]);

const Title = ({ children }) => (
  <View style={styles.titleContainer}>
    <Text style={Text.types.sectionTitle}>{children}</Text>
  </View>
);
Title.propTypes = {
  children: PropTypes.any,
};

const CallNoteV2Detail = ({
  onBackPress,
  call,
  callNote: {
    callResult,
    objectiveAchieved,
    rating,
    reflection,
    mood,
  },
  activity,
}) => (
  <BaseView>
    <Header>
      <Text style={Text.types.title}>
        {moment(call.startTime).format('ddd, MMM D, h:MM a')}
      </Text>
      <HeaderIcon
        uid="back"
        type={HeaderIcon.types.backDark}
        onPress={onBackPress}
      />
    </Header>

    <ScrollView style={styles.scrollView}>

      <View style={styles.section}>
        <Title>How Did the Call Go?</Title>

        <View style={styles.sectionBody}>
          <Text style={Text.types.paragraph}>
            {callResults[callResult]}
          </Text>
        </View>
      </View>

      {activity && (
        <View style={styles.section}>
          <Title>Activity</Title>

          <View style={styles.sectionBody}>
            <View style={styles.sectionBodyActivityItems}>
              {
                activity.icon.exists() && <Image
                  source={activity.icon.resize(72, 72).toSource()}
                  style={styles.activityImage}
                />
              }

              <Text style={[Text.types.paragraph, styles.activityObjective]}>
                {activity.objective}
              </Text>
            </View>
          </View>
        </View>
      )}

      <View style={styles.section}>
        <Title>Objective</Title>

        <View style={styles.sectionBody}>
          <Text style={Text.types.paragraph}>
            {objectAchievedOptions[objectiveAchieved]}
          </Text>
        </View>
      </View>

      <View style={styles.section}>
        <Title>Activity Rating</Title>

        <View style={styles.sectionBody}>
          <Text style={Text.types.paragraph}>
            {activityRatingOptions[rating]}
          </Text>
        </View>
      </View>

      <View style={styles.section}>
        <Title>Your Reflections</Title>

        <View style={styles.sectionBody}>
          <Text style={Text.types.paragraph}>
            {reflection}
          </Text>
        </View>
      </View>

      <View style={styles.section}>
        <Title>Your Menteeâ€™s Mood</Title>

        <View style={styles.sectionMoodBody}>
          <Image
            style={styles.menteeStateImage}
            source={mentee[mood.toLowerCase()]}
          />

          <Text style={Text.types.paragraph}>
            {upperFirst(mood.toLowerCase())}
          </Text>
        </View>
      </View>

     </ScrollView>
  </BaseView>
);


CallNoteV2Detail.propTypes = {
  onBackPress: PropTypes.func.isRequired,
  callNote: PropTypes.object,
  call: PropTypes.object,
  activity: PropTypes.object,
};


export default CallNoteV2Detail;
