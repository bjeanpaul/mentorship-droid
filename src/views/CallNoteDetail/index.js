import moment from 'moment';
import { fromPairs } from 'lodash';

import React, { PropTypes } from 'react';
import { View, ScrollView, Image } from 'react-native';
import { BaseView, Header, HeaderIcon, Text } from 'src/components';
import * as constants from 'src/constants/callNotes';
import images from 'src/constants/images';
import styles from './styles';


const mentee = fromPairs([
  [constants.CALL_NOTES_MENTEE_HAPPY, images.CALL_NOTES_MENTEE_HAPPY],
  [constants.CALL_NOTES_MENTEE_SAD, images.CALL_NOTES_MENTEE_SAD],
  [constants.CALL_NOTES_MENTEE_BORED, images.CALL_NOTES_MENTEE_BORED],
  [constants.CALL_NOTES_MENTEE_CONFUSED, images.CALL_NOTES_MENTEE_CONFUSED],
  [constants.CALL_NOTES_MENTEE_UPSET, images.CALL_NOTES_MENTEE_UPSET],
  [constants.CALL_NOTES_MENTEE_WITHDRAWN, images.CALL_NOTES_MENTEE_WITHDRAWN],
]);


const Title = ({ children }) => (
  <View style={styles.titleContainer}>
    <Text style={Text.types.sectionTitle}>{children}</Text>
  </View>
);
Title.propTypes = {
  children: PropTypes.any,
};

const CallNoteDetail = ({
  onBackPress,
  callNote: {
    activityHelpful,
    objectiveAchieved,
    menteeState,
    reflection,
    callStartTime,
  },
  activity,
}) => (
  <BaseView>
    <Header>
      <Text style={Text.types.title}>
        {moment(callStartTime).format('ddd, MMM D, h:MM a')}
      </Text>
      <HeaderIcon
        uid="back"
        type={HeaderIcon.types.backDark}
        onPress={onBackPress}
      />
    </Header>

    <ScrollView style={styles.scrollView}>
      {activity && (
        <View style={styles.section}>
          <Title>Discussion With Activity</Title>

          <View style={[styles.sectionBody, styles.sectionBodyActivity]}>
            {
              activity.icon.exists() && <Image
                source={activity.icon.toSource()}
                style={styles.activityImage}
              />
            }

            <Text style={[Text.types.paragraph, styles.activityObjective]}>
              {activity.objective}
            </Text>
          </View>
        </View>
      )}

      <View style={styles.section}>
        <Title>Your Reflections</Title>
        <Text style={Text.types.paragraph}>
          {reflection}
        </Text>
      </View>

      <View style={styles.section}>
        <Title>Your Menteeâ€™s Mood</Title>

        <View style={styles.sectionBody}>
          <Image
            style={styles.menteeStateImage}
            source={mentee[menteeState]}
          />
        </View>
      </View>

      {activity && (
        <View style={styles.section}>
          <Title>Objective Achieved</Title>

          <View style={styles.sectionBody}>
            <Image
              style={styles.objectiveAchievedImage}
              source={
                objectiveAchieved
                  ? images.CALL_NOTES_COMPLETED_YES_SELECTED
                  : images.CALL_NOTES_COMPLETED_NO_SELECTED
              }
            />
          </View>
        </View>
      )}

      <View style={styles.section}>
        <Title>Helpfulness of Activity</Title>
        <Text style={[Text.types.paragraph, styles.activityHelpfulText]}>
          {constants.RATING_ITEMS[activityHelpful]}
        </Text>
      </View>
    </ScrollView>
  </BaseView>
);


CallNoteDetail.propTypes = {
  onBackPress: PropTypes.func.isRequired,
  callNote: PropTypes.object,
  activity: PropTypes.object,
};


export default CallNoteDetail;
