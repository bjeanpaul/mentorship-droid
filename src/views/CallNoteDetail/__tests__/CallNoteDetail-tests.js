import React from 'react';
import { noop } from 'lodash';
import CallNoteDetail from 'src/views/CallNoteDetail';
import { uidEquals, fakeCallNote, fakeActivity } from 'app/scripts/helpers';

describe('CallNoteDetail', () => {
  const createComponent = (props = {}) => (
    <CallNoteDetail
      callNote={fakeCallNote()}
      activity={fakeActivity()}
      onBackPress={noop}
      {...props}
    />
  );

  it('should render', () => {
    const el = render(createComponent());
    expect(el).toMatchSnapshot();
  });

  it('should exclude the activity views if they are undefined', () => {
    const el = render(createComponent({ activity: void 0 }));
    expect(el).toMatchSnapshot();
  });

  it('should call onBackPress', () => {
    const onBackPress = jest.fn();
    const el = shallow(createComponent({ onBackPress }));

    el.findWhere(uidEquals('back'))
      .simulate('press');

    expect(onBackPress).toBeCalled();
  });
});
