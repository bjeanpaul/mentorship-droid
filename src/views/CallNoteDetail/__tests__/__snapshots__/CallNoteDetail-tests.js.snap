exports[`CallNoteDetail should exclude the activity icon if it does not exist 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      },
      undefined,
    ]
  }>
  <View
    style={
      Array [
        Object {
          "height": 72,
          "justifyContent": "center",
          "paddingLeft": 16,
          "paddingRight": 16,
        },
        Object {},
      ]
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#003035",
            "fontFamily": "Brandon Text",
          },
          Object {
            "fontSize": 13,
            "textAlign": "center",
          },
          Object {
            "fontFamily": "Brandon Text_medium",
            "fontSize": 20,
            "fontWeight": "400",
          },
        ]
      }>
      Wed, Sep 28, 7:09 pm
    </Text>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#ffbcbc",
            "borderColor": "red",
            "borderWidth": 1,
            "height": 100,
            "justifyContent": "center",
            "margin": 10,
            "width": 300,
          },
          undefined,
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#333333",
            "margin": 20,
          }
        }>
        TouchableNativeFeedback is not supported on this platform!
      </Text>
    </View>
  </View>
  <ScrollView
    style={
      Object {
        "padding": 24,
      }
    }>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          DISCUSSION WITH ACTIVITY
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "marginTop": 18,
            },
            Object {
              "flexDirection": "row",
            },
          ]
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Array [
                Object {
                  "fontSize": 18,
                  "paddingBottom": 12,
                  "paddingTop": 18,
                  "textAlign": "left",
                },
                Object {
                  "alignSelf": "center",
                  "paddingBottom": 0,
                  "paddingTop": 0,
                },
              ],
            ]
          }>
          raaar
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          YOUR REFLECTIONS
        </Text>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#003035",
              "fontFamily": "Brandon Text",
            },
            Object {
              "fontSize": 13,
              "textAlign": "center",
            },
            Object {
              "fontSize": 18,
              "paddingBottom": 12,
              "paddingTop": 18,
              "textAlign": "left",
            },
          ]
        }>
        It went well
      </Text>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          YOUR MENTEEâ€™S MOOD
        </Text>
      </View>
      <View
        style={
          Object {
            "marginTop": 18,
          }
        }>
        <Image
          source={1}
          style={
            Object {
              "alignSelf": "center",
            }
          } />
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          OBJECTIVE ACHIEVED
        </Text>
      </View>
      <View
        style={
          Object {
            "marginTop": 18,
          }
        }>
        <Image
          source={1}
          style={
            Object {
              "alignSelf": "center",
            }
          } />
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          HELPFULNESS OF ACTIVITY
        </Text>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#003035",
              "fontFamily": "Brandon Text",
            },
            Object {
              "fontSize": 13,
              "textAlign": "center",
            },
            Array [
              Object {
                "fontSize": 18,
                "paddingBottom": 12,
                "paddingTop": 18,
                "textAlign": "left",
              },
              Object {
                "textAlign": "center",
              },
            ],
          ]
        } />
    </View>
  </ScrollView>
</View>
`;

exports[`CallNoteDetail should exclude the activity views if they are undefined 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      },
      undefined,
    ]
  }>
  <View
    style={
      Array [
        Object {
          "height": 72,
          "justifyContent": "center",
          "paddingLeft": 16,
          "paddingRight": 16,
        },
        Object {},
      ]
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#003035",
            "fontFamily": "Brandon Text",
          },
          Object {
            "fontSize": 13,
            "textAlign": "center",
          },
          Object {
            "fontFamily": "Brandon Text_medium",
            "fontSize": 20,
            "fontWeight": "400",
          },
        ]
      }>
      Wed, Sep 28, 7:09 pm
    </Text>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#ffbcbc",
            "borderColor": "red",
            "borderWidth": 1,
            "height": 100,
            "justifyContent": "center",
            "margin": 10,
            "width": 300,
          },
          undefined,
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#333333",
            "margin": 20,
          }
        }>
        TouchableNativeFeedback is not supported on this platform!
      </Text>
    </View>
  </View>
  <ScrollView
    style={
      Object {
        "padding": 24,
      }
    }>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          YOUR REFLECTIONS
        </Text>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#003035",
              "fontFamily": "Brandon Text",
            },
            Object {
              "fontSize": 13,
              "textAlign": "center",
            },
            Object {
              "fontSize": 18,
              "paddingBottom": 12,
              "paddingTop": 18,
              "textAlign": "left",
            },
          ]
        }>
        It went well
      </Text>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          YOUR MENTEEâ€™S MOOD
        </Text>
      </View>
      <View
        style={
          Object {
            "marginTop": 18,
          }
        }>
        <Image
          source={1}
          style={
            Object {
              "alignSelf": "center",
            }
          } />
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          HELPFULNESS OF ACTIVITY
        </Text>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#003035",
              "fontFamily": "Brandon Text",
            },
            Object {
              "fontSize": 13,
              "textAlign": "center",
            },
            Array [
              Object {
                "fontSize": 18,
                "paddingBottom": 12,
                "paddingTop": 18,
                "textAlign": "left",
              },
              Object {
                "textAlign": "center",
              },
            ],
          ]
        } />
    </View>
  </ScrollView>
</View>
`;

exports[`CallNoteDetail should render 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      },
      undefined,
    ]
  }>
  <View
    style={
      Array [
        Object {
          "height": 72,
          "justifyContent": "center",
          "paddingLeft": 16,
          "paddingRight": 16,
        },
        Object {},
      ]
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#003035",
            "fontFamily": "Brandon Text",
          },
          Object {
            "fontSize": 13,
            "textAlign": "center",
          },
          Object {
            "fontFamily": "Brandon Text_medium",
            "fontSize": 20,
            "fontWeight": "400",
          },
        ]
      }>
      Wed, Sep 28, 7:09 pm
    </Text>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#ffbcbc",
            "borderColor": "red",
            "borderWidth": 1,
            "height": 100,
            "justifyContent": "center",
            "margin": 10,
            "width": 300,
          },
          undefined,
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#333333",
            "margin": 20,
          }
        }>
        TouchableNativeFeedback is not supported on this platform!
      </Text>
    </View>
  </View>
  <ScrollView
    style={
      Object {
        "padding": 24,
      }
    }>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          DISCUSSION WITH ACTIVITY
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "marginTop": 18,
            },
            Object {
              "flexDirection": "row",
            },
          ]
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Array [
                Object {
                  "fontSize": 18,
                  "paddingBottom": 12,
                  "paddingTop": 18,
                  "textAlign": "left",
                },
                Object {
                  "alignSelf": "center",
                  "paddingBottom": 0,
                  "paddingTop": 0,
                },
              ],
            ]
          }>
          raaar
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          YOUR REFLECTIONS
        </Text>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#003035",
              "fontFamily": "Brandon Text",
            },
            Object {
              "fontSize": 13,
              "textAlign": "center",
            },
            Object {
              "fontSize": 18,
              "paddingBottom": 12,
              "paddingTop": 18,
              "textAlign": "left",
            },
          ]
        }>
        It went well
      </Text>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          YOUR MENTEEâ€™S MOOD
        </Text>
      </View>
      <View
        style={
          Object {
            "marginTop": 18,
          }
        }>
        <Image
          source={1}
          style={
            Object {
              "alignSelf": "center",
            }
          } />
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          OBJECTIVE ACHIEVED
        </Text>
      </View>
      <View
        style={
          Object {
            "marginTop": 18,
          }
        }>
        <Image
          source={1}
          style={
            Object {
              "alignSelf": "center",
            }
          } />
      </View>
    </View>
    <View
      style={
        Object {
          "marginBottom": 45,
        }
      }>
      <View
        style={
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#dfe5e6",
            "borderStyle": "solid",
            "paddingBottom": 19,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#003035",
                "fontFamily": "Brandon Text",
              },
              Object {
                "fontSize": 13,
                "textAlign": "center",
              },
              Object {
                "color": "#9fb1b3",
                "fontFamily": "Brandon Text_bold",
                "fontSize": 12,
                "fontWeight": "bold",
                "letterSpacing": 1,
                "textAlign": "left",
              },
            ]
          }>
          HELPFULNESS OF ACTIVITY
        </Text>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#003035",
              "fontFamily": "Brandon Text",
            },
            Object {
              "fontSize": 13,
              "textAlign": "center",
            },
            Array [
              Object {
                "fontSize": 18,
                "paddingBottom": 12,
                "paddingTop": 18,
                "textAlign": "left",
              },
              Object {
                "textAlign": "center",
              },
            ],
          ]
        } />
    </View>
  </ScrollView>
</View>
`;
